packages/sn-controls-react/src/viewcontrols/NewView.tsx

```diff
@@ -3,28 +3,24 @@
  *
  */
 import React, { Component, createElement } from 'react'
-import { connect } from 'react-redux'
 
 import Button from '@material-ui/core/Button'
 import Grid from '@material-ui/core/Grid'
 import Typography from '@material-ui/core/Typography'
 import { Repository } from '@sensenet/client-core'
 import { ControlSchema } from '@sensenet/control-mapper'
 import { GenericContent, Schema } from '@sensenet/default-content-types'
-import { Actions, Reducers } from '@sensenet/redux'
 import MediaQuery from 'react-responsive'
 import { ReactClientFieldSettingProps } from '../fieldcontrols/ClientFieldSetting'
 import { reactControlMapper } from '../ReactControlMapper'
-import { RootStateType } from './index'
```

Comment:
The `RootStateType` is not needed in the index.ts file anymore. Also don't forget to remove the import from sensenet-redux there as well.